include/sys/zio_impl.h

```diff
@@ -24,7 +24,7 @@
  */
 
 /*
- * Copyright (c) 2013 by Delphix. All rights reserved.
+ * Copyright (c) 2012, 2015 by Delphix. All rights reserved.
```

Comment:
Not certain how it's in this case but I also handled it that way - if e.g. upstream commits from Illumos did it

@behlendorf @ahrens  would be good to know how the general policy is on how to proceed on these cases